test/hotspot/jtreg/compiler/c1/TestPinnedIntrinsics.java

```diff
@@ -25,7 +25,7 @@
  * @test
```

Comment:
Should we need to update the copyright year for the touch files

Reply:
`After the review, I will update the copyrights.`

It is IMO easier to review big changes without the noise.